### DISCUSSION TITLE
Webinar Support?

### DISCUSSION THREAD

**Discussion by waycos** [edumeet/edumeet#819] [Ideas]
Is there a study where only one person's camera is turned on and other participants' cameras are turned off? This is important for webinars. Is there such support?

**Comment by Aleeeeeeeex** [comment]
I am creating a "webinar" type with:

- #668 
- autoMuteThreshold    : 0
- lastN : 1
- lockLastN : true
- Hidden media controls as default so that no one can enable mic or camera.

This way i think it would work like a webinar.

**Comment by mi4aux** [comment]
- set one presenter (like suggested by [misi](https://github.com/edumeet/edumeet/issues/566#issuecomment-698202037))

and depending on the amount of users, also recommended: 

- showNotifications : false
- notificationSounds : false

**Comment by varundabke** [comment]
This can be done by 'fixing' the selected spotlight for everyone. Moderator can choose the fixed user and put everyone in Filmstrip mode remotely.
